[mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/10546/steps/compile__2/logs/stdio) `env ./mach build-cef`
```
--- stderr
indent: Command line: unknown parameter "--version"
No module named mako
configure: error: Package requirements (zlib >= 1.2.3) were not met:

No package 'zlib' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables ZLIB_CFLAGS
and ZLIB_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
thread 'main' panicked at 'failed with: exit code: 1', /Users/servo/.cargo/git/checkouts/osmesa-src-bf84ce2b80210dd7/6d23dae/build.rs:58:8
stack backtrace:
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -20,11 +20,11 @@ matrix:
         - export LLVM_CONFIG=/usr/lib/llvm-3.9/bin/llvm-config
         - export CC=gcc-5 CXX=g++-5
       script:
-         - ./mach build -d --verbose
-         - ./mach test-unit
+         - RUSTFLAGS='-D warnings' ./mach build -d --verbose
+         - RUSTFLAGS='-D warnings' ./mach test-unit
          - ./mach clean
-         - ./mach build-geckolib
-         - ./mach test-stylo
+         - RUSTFLAGS='-D warnings' ./mach build-geckolib
+         - RUSTFLAGS='-D warnings' ./mach test-stylo
          - bash etc/ci/lockfile_changed.sh
       cache:
         directories:

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -48,6 +48,8 @@ cache:
 #  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 
 build_script:
-  - mach build -d -v && mach test-unit
+  - set RUSTFLAGS=-D warnings
+  - mach build -d -v
+  - mach test-unit
 
 test: off

--- a/components/hashglobe/src/alloc.rs
+++ b/components/hashglobe/src/alloc.rs
@@ -28,6 +28,7 @@ pub use self::platform::{alloc, dealloc, realloc};
 mod platform {
     extern crate libc;
 
+    #[cfg(not(any(target_os = "android")))]
     use std::ptr;
 
     use super::MIN_ALIGN;

--- a/etc/ci/buildbot_steps.yml
+++ b/etc/ci/buildbot_steps.yml
@@ -32,11 +32,11 @@ mac-rel-wpt4:
 
 mac-dev-unit:
   - ./mach clean-nightlies --keep 3 --force
-  - env PKG_CONFIG_PATH=/usr/local/opt/zlib/lib/pkgconfig ./mach build --dev
-  - env ./mach test-unit
+  - env PKG_CONFIG_PATH=/usr/local/opt/zlib/lib/pkgconfig RUSTFLAGS=-Dwarnings ./mach build --dev
+  - env PKG_CONFIG_PATH=/usr/local/opt/zlib/lib/pkgconfig RUSTFLAGS=-Dwarnings ./mach test-unit
   - env ./mach package --dev
-  - env ./mach build-cef
-  - ./mach build-geckolib
+  - env PKG_CONFIG_PATH=/usr/local/opt/zlib/lib/pkgconfig RUSTFLAGS=-Dwarnings ./mach build-cef
+  - env RUSTFLAGS=-Dwarnings ./mach build-geckolib
   - bash ./etc/ci/lockfile_changed.sh
   - bash ./etc/ci/manifest_changed.sh
 
@@ -80,13 +80,13 @@ linux-dev:
   - ./mach clean-nightlies --keep 3 --force
   - ./mach test-tidy --no-progress --all
   - ./mach test-tidy --no-progress --self-test
-  - env CC=gcc-5 CXX=g++-5 ./mach build --dev
-  - env ./mach test-unit
+  - env CC=gcc-5 CXX=g++-5 RUSTFLAGS=-Dwarnings ./mach build --dev
+  - env CC=gcc-5 CXX=g++-5 RUSTFLAGS=-Dwarnings ./mach test-unit
   - env ./mach package --dev
-  - env ./mach build-cef
-  - env ./mach build --dev --no-default-features --features default-except-unstable
-  - ./mach build-geckolib
-  - ./mach test-stylo
+  - env CC=gcc-5 CXX=g++-5 RUSTFLAGS=-Dwarnings ./mach build-cef
+  - env CC=gcc-5 CXX=g++-5 RUSTFLAGS=-Dwarnings ./mach build --dev --no-default-features --features default-except-unstable
+  - env RUSTFLAGS=-Dwarnings ./mach build-geckolib
+  - env RUSTFLAGS=-Dwarnings ./mach test-stylo
   - bash ./etc/ci/lockfile_changed.sh
   - bash ./etc/ci/manifest_changed.sh
   - bash ./etc/ci/check_no_panic.sh
@@ -120,7 +120,7 @@ linux-nightly:
 
 android:
   - ./mach clean-nightlies --keep 3 --force
-  - env ANDROID_SDK=/home/servo/android/sdk/r25.2.3 ./mach build --android --dev
+  - env ANDROID_SDK=/home/servo/android/sdk/r25.2.3 RUSTFLAGS=-Dwarnings ./mach build --android --dev
   - env ANDROID_SDK=/home/servo/android/sdk/r25.2.3 ./mach package --android --dev
   - bash ./etc/ci/lockfile_changed.sh
   - bash ./etc/ci/manifest_changed.sh
@@ -134,18 +134,19 @@ android-nightly:
 
 arm32:
   - ./mach clean-nightlies --keep 3 --force
-  - env ./mach build --rel --target=arm-unknown-linux-gnueabihf
+  - env RUSTFLAGS=-Dwarnings ./mach build --rel --target=arm-unknown-linux-gnueabihf
   - bash ./etc/ci/lockfile_changed.sh
   - bash ./etc/ci/manifest_changed.sh
 
 arm64:
   - ./mach clean-nightlies --keep 3 --force
-  - env ./mach build --rel --target=aarch64-unknown-linux-gnu
+  - env RUSTFLAGS=-Dwarnings ./mach build --rel --target=aarch64-unknown-linux-gnu
   - bash ./etc/ci/lockfile_changed.sh
   - bash ./etc/ci/manifest_changed.sh
 
 windows-msvc-dev:
   - mach.bat clean-nightlies --keep 3 --force
+  - set RUSTFLAGS=-D warnings
   - mach.bat build --dev
   - mach.bat test-unit
   - mach.bat package --dev

--- a/ports/glutin/lib.rs
+++ b/ports/glutin/lib.rs
@@ -7,7 +7,6 @@
 
 #[macro_use] extern crate bitflags;
 extern crate compositing;
-#[cfg(target_os = "android")] extern crate egl;
 extern crate euclid;
 extern crate gleam;
 extern crate glutin;

